src/store/context-menu/context-menu-actions.ts

```diff
@@ -87,13 +88,10 @@ export const openProcessContextMenu = (event: React.MouseEvent<HTMLElement>) =>
     (dispatch: Dispatch, getState: () => RootState) => {
         const { location } = getState().router;
```

Comment:
why not simply: `const location = getState().router.location`?

Reply:
IMO simpler is to write location once rather than twice.

Reply:
OK It's a matter of personal preference actually :)